### DISCUSSION TITLE
Finn compiler for KV260?

### DISCUSSION THREAD

**Discussion by bestamigunay** [Xilinx/finn#512] [Q&A] [answered]
Hello, as you know, PYNQ for Kria KV260 just released. Kria board is supported by the Vivado 2021.1 or later versions. However, Finn requires Vivado 2020.1. So, it looks like we cannot synthesize a bitfile for Kria by using Finn compiler. Maybe I am wrong, is there any solution to use Finn compiler for Kria board? Thank you!

**Comment by nhma20** [comment, marked as answer]
Perhaps you can switch to the "feature/vitis_hls" or "feature/vitis_hls_2021.2" branches of the FINN repository to use the version of Vivado that you need
